src/utils/beamer.ts

```diff
@@ -0,0 +1,33 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-nocheck
+import { BEAMER_ID } from './constants'
+
+export const loadBeamer = async (): Promise<void> => {
+  const APP_ID = BEAMER_ID
+  if (!APP_ID) {
+    console.error('[Beamer] - In order to use Beamer you need to add an appID')
+    return
+  }
+
+  window.beamer_config = {
```

Comment:
Can you not initialise beamer with thes instead of modifying the `window` object?